tests/get-pod-test

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+log() {
+  _fg=''
+  [ "${1}" = "red" ] && _fg='\033[0;31m'
+  [ "${1}" = "green" ] && _fg='\033[0;32m'
+  shift
+  
+  echo "${_fg}>>> ${@}\033[0m"
+}
+
+
+namespace="${1}"
+github_team="${2}"
+
+if kubectl auth can-i get pods --namespace "${namespace}" --as=test --as-group=github:"${github_team}" | grep  -q "yes"; then
+    log green "you have access to this pod in this namespace"
```

Comment:
The colours aren't working for me in my terminal, not sure if that is my settings or what... 

![screen shot 2018-09-28 at 17 53 55](https://user-images.githubusercontent.com/3952074/46222220-85d19780-c347-11e8-8465-15f3c84dc85c.png)

Ignore the error for a mo (that's coz I don't have a user "test" I think), should that output be coloured? Might be that I am overriding it in my `bash_profile`... 🤔